### DISCUSSION TITLE
checkpoint request

### DISCUSSION THREAD

**Discussion by zzzzz314314** [fvmassoli/cross-resolution-face-recognition#15] [General]
Hello. Thank you for your work and sharing it. 
I tried to trained with your exact default hyperparameters in the code (also a third of your LR and triple your LR). After around 40 epochs, the validation on 8*8 seemed to converge to 35% acc. I tested them with Tinyface and have a r1 accuracy at most around 30% while it is reported 58% on the paper. 
I do researches in low resolution face recognition and experiment methods only for research use. I would be most grateful if you could try to share a checkpoint that is close to the one reported on the paper, please. Some details including training process and official request including signatures from my professors was sent to your email (my email: nn.cs09@nycu.edu.tw). Thank you.
